From 955cfdd91676f35635e6a55da513f64b7c5555c9 Mon Sep 17 00:00:00 2001
From: LI Qingwu <Qing-wu.Li@leica-geosystems.com.cn>
Date: Mon, 30 Sep 2024 11:54:19 +0800
Subject: [PATCH 3/4] WPE Platform: fix wpe_toplevel_qtquick has not been
 declared

Fix the following compile failed issue for Qt6:

Source/WebKit/UIProcess/API/wpe/qt6/WPEToplevelQtQuick.cpp:35:46:
error: 'wpe_toplevel_qtquick' has not been declared

Upstream-Status: Submitted [https://github.com/WebKit/WebKit/pull/34441]

Signed-off-by: LI Qingwu <Qing-wu.Li@leica-geosystems.com.cn>
---
 Source/WebKit/UIProcess/API/wpe/qt6/WPEToplevelQtQuick.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Source/WebKit/UIProcess/API/wpe/qt6/WPEToplevelQtQuick.cpp b/Source/WebKit/UIProcess/API/wpe/qt6/WPEToplevelQtQuick.cpp
index ff6daf3e9a28..966734abefc4 100644
--- a/Source/WebKit/UIProcess/API/wpe/qt6/WPEToplevelQtQuick.cpp
+++ b/Source/WebKit/UIProcess/API/wpe/qt6/WPEToplevelQtQuick.cpp
@@ -26,6 +26,8 @@
 #include "config.h"
 #include "WPEToplevelQtQuick.h"
 
+#include <wtf/glib/WTFGType.h>
+
 /**
  * WPEToplevelQtQuick:
  *
-- 
2.34.1

